cdk/lib/cache.ts

```diff
@@ -1,26 +1,34 @@
-import { CfnOutput, Construct, Stack } from '@aws-cdk/core';
-import { CfnCacheCluster, CfnSubnetGroup } from '@aws-cdk/aws-elasticache';
-import { Alarm, Metric } from '@aws-cdk/aws-cloudwatch';
-import { Peer, Port, SecurityGroup, Vpc } from '@aws-cdk/aws-ec2';
+import {
+  aws_cloudwatch as cloudwatch,
+  aws_ec2 as ec2,
+  aws_elasticache as elasticache,
+  Stack,
+  CfnOutput,
+} from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
 import { CacclAppEnvironment } from './appEnvironment';
 
 export interface CacclCacheOptions {
-  engine: string,
-  numCacheNodes?: number,
-  cacheNodeType?: string,
-};
+  engine: string;
+  numCacheNodes?: number;
+  cacheNodeType?: string;
+}
 
 export interface CacclCacheProps {
```

Comment:
There are lots of these interfaces at the top of functions in this project. Do you think it'd be a good idea to separate them into their own file? Not sure if that helps or hurts organization.